frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/auto_completion_node_widget.dart

```diff
@@ -204,6 +208,15 @@ class _AutoCompletionInputState extends State<_AutoCompletionInput> {
     );
   }
 
+  Future<void> _updateGenerationCount() async {
+    final transaction = widget.editorState.transaction;
+    transaction.updateNode(widget.node, {
+      kAutoCompletionGenerationCount:
+          widget.node.attributes[kAutoCompletionGenerationCount] + 1
```

Comment:
Why do we need to this parameter? `kAutoCompletionGenerationCount`

Reply:
we need this to check how many times the response is being generated. Because I thought we need to show the input field only when the user generates the response for the first time and after that we don't show text field.